docs/zh-CN/benchmark/ssb-and-flat.md

```diff
@@ -0,0 +1,233 @@
+---
+{
+    "title": "Star-Schema-Benchmark 测试",
+    "language": "zh-CN"
```

Comment:
Why do Chinese and English SSB test documents not correspond?

Reply:
I just provide a basic version, and Jiafeng will improve the documentation next. This document may be displayed to users, and Jiafeng is better at writing documents.